jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -1060,11 +1073,13 @@ def calculate_batch_photometry(self, options=[], add_to_table=True, update_plots
                 option.setdefault('pixel_area', defaults.get('pixel_area', 0))
             if self.flux_scaling_multi_auto:
                 option.setdefault('flux_scaling', defaults.get('flux_scaling', 0))
+
             try:
                 self.calculate_photometry(add_to_table=add_to_table,
                                           update_plots=this_update_plots,
                                           **option)
             except Exception as e:
+                raise

```

Comment:
Is this debugging leftover?
```suggestion
```

Reply:
Good catch, I was debugging the error the added test uncovered.